### ISSUE TITLE
Where is convert_bert_original_tf_checkpoint_to_pytorch.py?

### ISSUE THREAD

**Issue by sednaasil** [huggingface/transformers#9556]
HI:  

I am getting the following error when implementing entity extraction in BERT.  OSError: Error no file named ['pytorch_model.bin', 'tf_model.h5', 'model.ckpt.index']

I am very new to using BERT, and noted that [issue 2110](https://github.com/huggingface/transformers/issues/2110) had a similar issue.  Issue 2110 was referred to the convert_bert_original_tf_checkpoint_to_pytorch.py file.  However, the current link isn't working.  Could you point me to its current location?

V/r,
L

**Comment by NielsRogge** [comment]
It's current location is [here](https://github.com/huggingface/transformers/blob/master/src/transformers/models/bert/convert_bert_original_tf_checkpoint_to_pytorch.py).

**Comment by LysandreJik** [comment]
Hi @sednaasil, what are you trying to do? Could you show the code that you're using so that we may help you debug it? Thanks.

**Comment by sednaasil** [comment]
Hi!  

I downloaded the uncased_L-12_H-768_A-12 BERT model to create an entity extraction tool following this [method](https://github.com/abhishekkrthakur/bert-entity-extraction).  The model I downloaded did not include 'pytorch_model.bin', 'tf_model.h5', 'model.ckpt.index' files, resulting in an error implementing my model.  I saw a previous post where the files I was missing were generated with the convert_bert_original_tf_checkpoint_to_pytorch.py file; however the link to the file was broken.  Is this the correct way to proceed?
